test/e2e/framework/util.go

```diff
@@ -4474,7 +4462,34 @@ func CheckConnectivityToHost(f *Framework, nodeName, podName, host string, timeo
 			Logf("pod %s/%s logs:\n%s", f.Namespace.Name, pod.Name, logs)
 		}
 	}
+	return err
+}
 
+// CheckConnectivityToHost launches a pod to test connectivity to the specified
+// host. An error will be returned if the host is not reachable from the pod.
+//
+// An empty nodeName will use the schedule to choose where the pod is executed.
```

Comment:
typo: schedule -> scheduler

Reply:
Thanks for your review, @caseydavenport 

A nice catch, that was the original typo by moving the comment. This is a good chance to fix it also.